hibernate-core/src/main/java/org/hibernate/procedure/internal/StandardCallableStatementSupport.java

```diff
@@ -47,86 +45,41 @@ public StandardCallableStatementSupport(boolean supportsRefCursors) {
 	public JdbcCall interpretCall(
 			String procedureName,
 			FunctionReturnImpl functionReturn,
-			ParameterMetadataImplementor parameterMetadata,
+			ProcedureParameterMetadataImplementor parameterMetadata,
 			ProcedureParamBindings paramBindings,
 			SharedSessionContractImplementor session) {
 		final StringBuilder buffer = new StringBuilder().append( "{call " )
```

Comment:
Using the perfect size estimate would be awesome :)

```suggestion
		final StringBuilder buffer = new StringBuilder(9 + procedureName.length() + registrations.size() * 2).append( "{call " )
```

Reply:
done :+1: